Controller/AdminSecurityController.php

```diff
@@ -12,29 +12,29 @@
 namespace Sonata\UserBundle\Controller;
 
 use FOS\UserBundle\Model\UserInterface;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\RedirectResponse;
```

Comment:
no problem, just a info @Soullivaneuh 

@dbu thanks for the info, so i'm :+1: too to keep this line!